src/Microcks.Testcontainers/MicrocksContainerExtensions.cs

```diff
@@ -232,28 +219,16 @@ public static Boolean Verify(this MicrocksContainer container, string serviceNam
     /// <param name="serviceVersion">Service version</param>
     /// <param name="invocationDate">Date of invocation</param>
     /// <returns>True if the service has been invoked at least once, false otherwise</returns>
-    public static Boolean Verify(this MicrocksContainer container, string serviceName, string serviceVersion, DateOnly? invocationDate)
+    public static async Task<Boolean> Verify(this MicrocksContainer container, string serviceName, string serviceVersion, DateOnly? invocationDate = null)
```

Comment:
```suggestion
    public static async Task<bool> VerifyAsync(this MicrocksContainer container, string serviceName, string serviceVersion, DateOnly? invocationDate = null)
```


Always prefer using bool. Reserve Boolean for cases where you need to access static methods. 
```
bool microcksIsAwesome = true;
string str = Boolean.ToString(microcksIsAwesome);
```